templates/terraform/modules/environment/main.tf

```diff
@@ -45,29 +45,45 @@ module "kube2iam" {
 
 # @TODO - Move this to a different file
 
-# {{ if .Config.Infrastructure.AWS.Cognito }}
-# ref: https://github.com/squidfunk/terraform-aws-cognito-auth#usage
-
-# data "aws_acm_certificate" "wildcard_cert" {
-#   domain   = "*.${var.public_dns_zone}"
-# }
-
-module "cognito-auth" {
```

Comment:
This module wasn't working?

Reply:
yah, had some issues. It was pretty complex and had a lot of moving parts. It never ran cleanly for me and also added some dependencies like `jq`

Reply:
😢